### PR TITLE
Heroku now defaults to Ruby 3.0 but Template is built with Ruby 2.7 

### PR DESCRIPTION
**Pull request by zzJZzz**
Updated gemfile to to Ruby 2.7
Updated readme.md to reflect the instructions.
Added a Procfile.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,54 +9,70 @@ This is a community maintained Starter Project Battlesnake!
...
+   ```
 
 3. [Create a new Heroku app](https://devcenter.heroku.com/articles/creating-apps) to run your Battlesnake.
-    ```shell
-    heroku create [YOUR-APP-NAME]
-    ```
+   ```shell
+   heroku create [YOUR-APP-NAME]
+   ```
+   **Important:** The Starter Snake is built to use Ruby 2.7, but Heroku will default to Ruby 3.0 now as of May 2022.
```
Comment (bvanvugt) [inline README.md]: Can you remove this note since we're doing this by default with this change?

File: app/app.rb [hunk 2]
```diff
@@ -11,10 +11,10 @@
 get '/' do
   appearance = {
     apiversion: "1",        
-    author: "",           # TODO: Your Battlesnake Username
-    color: "#888888",     # TODO: Personalize
-    head: "default",      # TODO: Personalize
-    tail: "default",      # TODO: Personalize
+    author: "JZ",           # TODO: Your Battlesnake Username
```
Comment (bvanvugt) [inline app/app.rb]: Please also remove your customizations from this PR :)

**Review by bvanvugt — CHANGES_REQUESTED** [review]
Thanks for this! I'd love to get this merged, just a couple comments we'd like you to address first 👍

File: README.md (hunk 1 above)
Comment (zzJZzz) [reply README.md]: Thank you. I wanted to confirm. Lines 37-40? Or line 40?

File: app/app.rb (hunk 2 above)
Comment (zzJZzz) [reply app/app.rb]: Whoops. Yes I'll get on this!

**Comment by zzJZzz** [comment]
@bvanvugt I am still an early stage developer but I believe I have it updated. Thank you for your patience.

**Comment by zzJZzz** [comment]
@bvanvugt  accidentally closed it

**Review by bvanvugt — APPROVED** [review]

**Pull request merged**